var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -167,6 +172,16 @@ def install(self, spec, prefix):
             os.makedirs(os.path.join(prefix, "src"))
             os.symlink(includedir, os.path.join(prefix, "include"))
 
+        if self.spec.satisfies('@:8.0.61'):
+            # Perl 5.26 removed current directory from module search path.
+            # We are addressing this by exporting `PERL5LIB` earlier, but for some
+            # reason, it is not enough. One more file needs to be extracted before
+            # running actual installer. This solution is one of the commonly found
+            # in the Internet, when people try to install CUDA <= 8 manually.
```

Comment:
If you have a stackoverflow link or so, feel free to cite it here
```suggestion
            # running the actual installer. This solution is one of the commonly
            # found on the Internet, when people try to install CUDA <= 8 manually.
```